Utils/trigger_private_build.py

```diff
@@ -0,0 +1,125 @@
+import re
+import os
+import sys
+import json
+import time
+import argparse
+import requests
+import logging
+import demisto_sdk.commands.common.tools as tools
+
+# disable insecure warnings
+requests.packages.urllib3.disable_warnings()
+
+INFRASTRUCTURE_FILES = ['Tests/scripts/validate_premium_packs.sh', 'Tests/scripts/validate_premium_packs.py',
+                        'Tests/scripts/validate_index.py']
+INFRASTRUCTURE_FOLDERS = ['Tests/private_build', 'Tests/Marketplace']
```

Comment:
```suggestion
PRIVATE_BUILD_INFRA_FOLDERS = ['Tests/private_build', 'Tests/Marketplace']
```
If you have better ideas I'm good with them as long as they are more accurate.